docs/translations/translations.json

```diff
@@ -66,7 +66,7 @@
   "copySourceLinkJS": "Copy link to JavaScript source",
   "copySourceLinkTS": "Copy link to TypeScript source",
   "stackblitz": "Edit in StackBlitz (JS only)",
-  "headTitle": "The world's most popular React UI framework - Material-UI",
+  "headTitle": "The world's second most popular React UI framework - Material-UI",
```

Comment:
What about going for simplicity? 
```suggestion
  "headTitle": "Material-UI",
```

Reply:
[Same idea](https://github.com/ant-design/ant-design/pull/17573#discussion_r302105013)

Otherwise we could setup a bot that monitors it and files PRs accordingly :smile:

Reply:
The wording orgnially comes from replicating some of Bootstrap SEO structure.

Reply:
https://github.com/ant-design/ant-design/pull/17573#issuecomment-510121410

Reply:
I wasn't all that serious. Would still change it though.

Reply:
Agree

Reply:
how about "the best"?